.github/workflows/unstable-release.yml

```diff
@@ -0,0 +1,31 @@
+name: Unstable Release
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  push-master:
+    # Don't run on any commit that features [skip ci] in the commit message
+    if: "! contains(github.event.head_commit.message, '[skip ci]')"
+    name: Build, test and publish unstable release
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [10.15.3]
```

Comment:
Not sure if it matters, but can we do v12?

Reply:
Changed